src/Microsoft.OData.Client/BatchSaveResult.cs

```diff
@@ -51,6 +51,9 @@ internal class BatchSaveResult : BaseSaveResult
         /// <summary>Buffer used for caching operation response body streams.</summary>
         private byte[] streamCopyBuffer;
 
+        /// <summary>If batch request is allowed to use Relative Uri.</summary>
+        private bool UseRelativeUri = false;

```

Comment:
>UseRelativeUri [](start = 21, length = 14)

nit: private members usually lower camel case (first letter is lower case). public/protected/internal are pascal (upper camel) case.

Reply:
I have fixed this